ansible/roles/openwrt-image/tasks/prepare.yml

```diff
@@ -13,7 +13,7 @@
     state: directory
     mode: '0755'
   with_items:
-    - "{{ openwrt_download_dir }}/dl"
+    - "{{ openwrt_download_dir }}/dl/{{ openwrt_arch }}"
```

Comment:
Nice  :+1:

Reply:
ok so i will move on with it. As you already mentioned above it doesn't hold all the info we want in there after all.